stable/jenkins/templates/config.yaml

```diff
@@ -214,9 +214,9 @@ data:
     </jenkins.CLI>
 {{- end }}
   apply_config.sh: |-
-{{- if .Values.master.enableXmlConfig }}
     mkdir -p /usr/share/jenkins/ref/secrets/;
     echo "false" > /usr/share/jenkins/ref/secrets/slave-to-master-security-kill-switch;
```

Comment:
I would move this inside the if statement. It's a useful configuration, but very similar to XML configs. So I'd prefer to make it via JCasC plugin in the future.

Reply:
The reason for moving it outside the if statement is that if you have a setting of `enableXmlConfig: false` and `secretsFileSecret` configured the startup will fail due to failed copy due to missing `/usr/share/jenkins/ref/secrets/` directory (line 289).
Will have to duplicate the mkdir then. Ok?

Reply:
`mkdir -p /usr/share/jenkins/ref/secrets/;` outside of if absolutely makes sense.

I was just talking about `echo "false" > /usr/share/jenkins/ref/secrets/slave-to-master-security-kill-switch;`